src/mui/button/SaveButton.js

```diff
@@ -49,6 +52,12 @@ SaveButton.propTypes = {
     raised: PropTypes.bool,
     saving: PropTypes.bool,
     translate: PropTypes.func.isRequired,
+    submitOnEnter: PropTypes.bool,
+    handleSubmit: PropTypes.func,
+};
+
+SaveButton.defaultProps = {
+    submitOnEnter: true,
```

Comment:
I'm not sure I understand you. But I agree that if there is a place to put the default, it's the form.